README.md

```diff
@@ -148,83 +78,115 @@ ray up deploy/ray/aviary-cluster.yaml
 ray attach deploy/ray/aviary-cluster.yaml
 
 # Deploy the LightGPT model. 
-aviary run --model ~/models/continuous_batching/amazon--LightGPT.yaml
+cd aviary && serve run serve/amazon--LightGPT.yaml
 ```
 
 You can deploy any model in the `models` directory of this repo, 
 or define your own model YAML file and run that instead.
 
 ### On Kubernetes
 
-For Kubernetes deployments, see [Aviary on GKE guide](https://ray-project.github.io/aviary/kuberay/deploy-on-gke/) and [Aviary on EKS guide](https://ray-project.github.io/aviary/kuberay/deploy-on-eks/).
+For Kubernetes deployments, please see our extensive documentation for [deploying Ray Serve on KubeRay](https://docs.ray.io/en/latest/serve/production-guide/kubernetes.html).
+## Query your models
 
-## Query Aviary
-
-Once the Aviary Backend is running, you can install the Aviary Client outside of the Docker container to query the backend.
+Once the models are deployed, you can install a client outside of the Docker container to query the backend.
 
 ```shell
-pip install "aviary @ git+https://github.com/ray-project/aviary.git"
+pip install "aviary @ git+https://github.com/ray-project/ray-llm.git"
 ```
 
-You can query Aviary in many ways. Here we outline 4. 
+You can query your RayLLM deployment in many ways.
 
 In all cases start out by doing: 
 
 ```shell
 export AVIARY_URL="http://localhost:8000/v1"
```

Comment:
can we change this? It's just defining an environment variable right?

Reply:
Yeah, what would you like to change it to?

Reply:
updated

Reply:
LLM_SERVE_URL?